### ISSUE TITLE
Publish on CRAN

### ISSUE THREAD

**Issue by trinhdhk** [moodymudskipper/nakedpipe#32]
Any plan to publish this on CRAN?

**Comment by moodymudskipper** [comment]
Yes, but I'm not coding a lot these days and have other things on my CRAN list. I'll get to it I promise, thanks for the interest!
